### PR TITLE
T2 quantum first release

### PR DESCRIPTION
**Pull request by 00Kai0**
https://github.com/Azure/sdk-release-request/issues/1142
--

### CODE REVIEW

File: sdk/quantum/azure-mgmt-quantum/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Microsoft Azure SDK for Python
+
+This is the Microsoft Azure MyService Management Client Library.
```
Comment (msyyc) [inline sdk/quantum/azure-mgmt-quantum/README.md:3]: Replace 'MyService' with right name

File: sdk/quantum/azure-mgmt-quantum/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Release History
+
+## 1.0.0b1 (1970-01-01)
```
Comment (jsntcy) [inline sdk/quantum/azure-mgmt-quantum/CHANGELOG.md:3]: Update date.

**Review by jsntcy — APPROVED** [review]

**Pull request merged**